test/runtests.jl

```diff
@@ -909,18 +909,47 @@ else  # run HiGHS tests
         end
 
         @testset "Minimize Unserved Load" begin
-                
+            d = JSON.parsefile("./scenarios/outage.json")
+            d["ElectricLoad"]["loads_kw"] = ones(8760)*200.0
+            d["ElectricLoad"]["loads_kw"][5100:5109] .= 410.0
+            d["ElectricLoad"]["loads_kw"][5200:5209] .= 410.0
+            d["ElectricLoad"]["critical_load_fraction"] = 0.5
+            d["PV"]["existing_kw"] = 0.0
+            d["PV"]["min_kw"] = 100.0
+            d["PV"]["max_kw"] = 100.0
+            d["CHP"]["min_kw"] = 100.0
+            d["CHP"]["max_kw"] = 100.0
+            d["Generator"]["existing_kw"] = 0.0
+            d["Generator"]["min_kw"] = 100.0
+            d["Generator"]["max_kw"] = 100.0
+            d["ElectricStorage"]["min_kw"] = 20
+            d["ElectricStorage"]["max_kw"] = 20
+            d["ElectricStorage"]["min_kwh"] = 50
+            d["ElectricStorage"]["max_kwh"] = 50
+            d["Financial"]["microgrid_upgrade_cost_fraction"] = 0.0
```

Comment:
@zolanaj the original test did have a `microgrid_upgrade_cost_fraction` in the post and I'm wondering if that's a good thing to test. Would including this cost slow down this test significantly?